packages/flutter/lib/src/foundation/internal_web.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The largest SMI value.
```

Comment:
Did you ever figure out how dartdocs deals with conditional imports?

Reply:
Currently it does not include the (dart.html) import, which is what we want. Should we put any doc comments on the unsupported implementation?